openmetadata-ui/src/main/resources/ui/src/components/MyAssetStats/MyAssetStats.component.tsx

```diff
@@ -73,10 +73,7 @@ const MyAssetStats: FunctionComponent<MyAssetStatsProps> = ({
         icon: Icons.TEST_SUITE,
         data: 'Test Suites',
         count: entityCounts.testSuiteCount,
-        link: getSettingPath(
-          GlobalSettingsMenuCategory.DATA_QUALITY,
-          GlobalSettingOptions.TEST_SUITE
-        ),
+        link: ROUTES.TEST_SUITES,
```

Comment:
I think we need to remove it from settings, Let's confirm it with @harshach

Reply:
@chirag-madlani , I have removed it from the settings.